templatetags/htk_tags.py

```diff
@@ -206,6 +206,14 @@ def get_django_setting(context, key):
         context[key] = value
     return ''
 
+@register.simple_tag()
+def get_htk_setting(key):
+    """Retrieves a Django setting and sets it on the context dictionary
+    """
+    from htk.utils import htk_setting
```

Comment:
change the import to

`from htk.utils import htk_setting as _htk_setting`

invoke it as `value = _htk_setting(key)`